.eslintrc

```diff
@@ -0,0 +1,224 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  // includes the typescript specific rules found here: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules
+  "plugins": [
+    "@typescript-eslint",
+    "import",
+    "no-only-tests",
+    "no-relative-import-paths",
+    "prettier"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier"
+  ],
+  "ignorePatterns": [
+    "package.json",
+    "./frontend/src/third_party/mlmd",
+    "./frontend/src/__tests__/cypress/.nyc_output",
+    "./frontend/src/__tests__/cypress/coverage",
+    "./frontend/package.json",
+    "./frontend/config"
```

Comment:
Can these stay in frontend or does it end up overriding the root level ignore patterns?